tests/src/FunctionalJavascript/ListPagesFiltersTest.php

```diff
@@ -255,9 +260,9 @@ public function testSelectedListPageFilters(): void {
     $this->assertSession()->pageTextContains('one yellow fruit');
     $this->assertSession()->pageTextContains('another yellow fruit');
     $this->assertSession()->elementExists('css', '.field--name-extra-field-oe-list-page-selected-filtersnodeoe-list-page');
-    $this->assertOptionSelected('Published', 'Yes');
-    $this->assertOptionSelected('Select one', 'test1');
-    $this->assertOptionSelected('Select one', 'test2');
+    $this->assertSession()->optionExists('Published', 'Yes');
+    $this->assertSession()->optionExists('Select one', 'test1');
+    $this->assertSession()->optionExists('Select one', 'test2');
```

Comment:
Checking of 'selected' attribute is missed.

Reply:
This will be fixed on next update.